plugins/wasi_nn/ggml.cpp

```diff
@@ -12,6 +12,20 @@
 
 namespace WasmEdge::Host::WASINN::GGML {
 #ifdef WASMEDGE_PLUGIN_WASI_NN_BACKEND_GGML
+llama_context_params wasmedge_llama_context_params() noexcept {
+  llama_context_params Params = llama_context_default_params();
+  const char *LlamaNContextEnv = std::getenv("LLAMA_N_CTX");
+  const char *LlamaLogEnv = std::getenv("LLAMA_LOG");
+  if (LlamaNContextEnv != nullptr) {
+    Params.n_ctx = std::stoi(LlamaNContextEnv);
```

Comment:
Should we cover these conversions in a try block? If the given n_ctx is invalid, what will happen?

Reply:
Added.